libraries/shared/src/shared/WebRTC.h

```diff
@@ -17,15 +17,15 @@
 #endif
 
 // WEBRTC_AUDIO: WebRTC audio features, e.g., echo canceling.
-// WEBRTC_DATA_CHANNEL: WebRTC client-server connections in parallel with UDP.
+// WEBRTC_DATA_CHANNELS: WebRTC client-server connections in parallel with UDP.
 
 #if defined(Q_OS_MAC)
 #  define WEBRTC_AUDIO 1
 #  define WEBRTC_POSIX 1
 #  define WEBRTC_LEGACY 1
 #elif defined(Q_OS_WIN)
 #  define WEBRTC_AUDIO 1
-#  define WEBRTC_DATA_CHANNEL 1
+#  define WEBRTC_DATA_CHANNELS 1
```

Comment:
Perhaps the constants should be named something like `ENABLE_WEBRTC_DATA_CHANNELS` instead, for clarity?

Reply:
I think the "topic" of the define ("WEBRTC") coming first makes it easier to distinguish these defines from other defines and group related ones together.